src/modules/endpoint/endpoints/locations.js

```diff
@@ -23,7 +23,7 @@ export default new EndpointBuilder('locations')
         code: stripSlashes(location.path),
         live: location.live,
         eventsEnabled: location['ige-evts'] === '1',
-        extrasEnabled: true
+        extrasEnabled: true // todo
```

Comment:
Is there already a task for this?

Reply:
Good idea, do you want to do it in this PR?

Reply:
this becomes obsolete with the use of the api v3 of the locations (cities) endpoint, which was already done in the WebApp, which isn't lots of work.

Reply:
is this even relevant for the Wohnungsbörse whether extras (and events) are enabled or not?